contracts/src/BeefyClient.sol

```diff
@@ -142,6 +147,9 @@ contract BeefyClient {
     ValidatorSet public currentValidatorSet;
     ValidatorSet public nextValidatorSet;
 
+    uint256[] currentValidatorSetCounters;
+    uint256[] nextValidatorSetCounters;
```

Comment:
Needs doc comments please.

Reply:
Addressed in https://github.com/Snowfork/snowbridge/pull/956/commits/17d468f0427d9e040b1d8438d9202045834b6fe5.